modules/nf-core/hifiasm/main.nf

```diff
@@ -11,114 +11,100 @@ process HIFIASM {
     tuple val(meta) , path(long_reads)        , path(ul_reads)
     tuple val(meta1), path(paternal_kmer_dump), path(maternal_kmer_dump)
     tuple val(meta2), path(hic_read1)         , path(hic_read2)
+    tuple val(meta3), path(bin_files)
 
     output:
-    tuple val(meta), path("*.r_utg.gfa")       , emit: raw_unitigs
-    tuple val(meta), path("*.ec.bin")          , emit: corrected_reads
-    tuple val(meta), path("*.ovlp.source.bin") , emit: source_overlaps
-    tuple val(meta), path("*.ovlp.reverse.bin"), emit: reverse_overlaps
-    tuple val(meta), path("*.bp.p_ctg.gfa")    , emit: processed_contigs, optional: true
-    tuple val(meta), path("*.p_utg.gfa")       , emit: processed_unitigs, optional: true
-    tuple val(meta), path("*.asm.p_ctg.gfa")   , emit: primary_contigs  , optional: true
-    tuple val(meta), path("*.asm.a_ctg.gfa")   , emit: alternate_contigs, optional: true
-    tuple val(meta), path("*.hap1.p_ctg.gfa")  , emit: paternal_contigs , optional: true
-    tuple val(meta), path("*.hap2.p_ctg.gfa")  , emit: maternal_contigs , optional: true
-    tuple val(meta), path("*.log")             , emit: log
-    path  "versions.yml"                       , emit: versions
+    tuple val(meta), path("*.r_utg.gfa")                             , emit: raw_unitigs
+    tuple val(meta), path("*.bin")                                   , emit: bin_files        , optional: true
+    tuple val(meta), path("*.p_utg.gfa")                             , emit: processed_unitigs, optional: true
+    tuple val(meta), path("${prefix}.{p_ctg,bp.p_ctg,hic.p_ctg}.gfa"), emit: primary_contigs  , optional: true
+    tuple val(meta), path("${prefix}.{a_ctg,hic.a_ctg}.gfa")         , emit: alternate_contigs, optional: true
+    tuple val(meta), path("${prefix}.*.hap1.p_ctg.gfa")              , emit: hap1_contigs     , optional: true
+    tuple val(meta), path("${prefix}.*.hap2.p_ctg.gfa")              , emit: hap2_contigs     , optional: true
+    tuple val(meta), path("*.ec.fa.gz")                              , emit: fasta            , optional: true
+    tuple val(meta), path("*.ovlp.paf.gz")                           , emit: paf              , optional: true
```

Comment:
Could you give names that tell about what's in files rather than their format ? e.g. `corrected_reads` for `fasta`, and `read_overlap` for `paf`.